### ISSUE TITLE
Remove deprecated t.trial.runner.DocTestSuite

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5111]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5111|
|Type|task|
|Created|2011-05-16 19:01:56Z|
|Branch|https://github.com/twisted/twisted/tree/remove-doctestsuite-5111|

`DocTestSuite` in [twisted/trial/runner.py](twisted/trial/runner.py) was deprecated in twisted 8.0 so it can be removed.

Attachments:

* [doctestsuite.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ec6/ec6de4c24a50e4788b61a9b4346f8f3315afc723/131095bed985ff3d131b74c380f2b0b61fab6824.patch) (3732 bytes) - added by thijs on 2011-05-16 19:02:23Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5111 5111
type__task task
reporter__thijs thijs
priority__lowest lowest
milestone__ 
branch__branches_remove_doctestsuite_5111 branches/remove-doctestsuite-5111
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__ 
time__1305572516000000 1305572516000000
changetime__1307310773000000 1307310773000000
version__None None
owner__thijs thijs
cc__thijs cc__jml cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess commented|
|-|-|

(In [32031]) Branching to 'remove-doctestsuite-5111'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Thanks for working on this, thijs. Looks good, please merge!

(Build results [here](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-doctestsuite-5111))

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [32047]) Merge remove-doctestsuite-5111: removed twisted.trial.runner.DocTestSuite, deprecated in Twisted 8.0.

Author: thijs
Reviewer: jesstess
Fixes: [#5111](https://github.com/twisted/twisted/issues/5111)
